app/controllers/warehouse_reports/health/cp_roster_controller.rb

```diff
@@ -9,12 +9,12 @@ class CpRosterController < ApplicationController
     before_action :require_can_administer_health!
 
     def index
-      @files = file_scope
+      @pagy, @files = pagy(file_scope)
```

Comment:
I'm not sure what you mean

Reply:
Trying to to decide if
```
@files = file_scope
@pagy, @files = pagy(@files)
```
Seems better. Because it also seems kinda silly since all it is doing is aliasing `file_scope`.
So, I'm leaving it for you to decide :-)

Reply:
oh okay, I'll leave as is. I committed your suggestions but won't go through the whole codebase to standardize calling `pagy` on instance variables-- chatted with Elliot and we agreed it probably wasnt worth the effort